driver-core/src/test/java/com/datastax/driver/core/querybuilder/QueryBuilderExecutionTest.java

```diff
@@ -182,4 +186,136 @@ public void should_delete_map_entry_with_bind_marker() throws Exception {
         assertThat(actual).containsExactly(entry(2, "bar"));
     }
 
+    /**
+     * Validates that {@link QueryBuilder} appropriately builds a select query using a tuple 'eq' comparison, i.e.:
```

Comment:
I think the correct jargon is "multi-column equal relation", but I'm fine with "tuple equal comparison".